viewer/packages/pointclouds/src/styling/PointCloudObjectCollection.ts

```diff
@@ -0,0 +1,28 @@
+/*!
+ * Copyright 2022 Cognite AS
+ */
+
+import assert from 'assert';
+
+import { EventTrigger } from '@reveal/utilities';
+
+/**
+ * Base class for collections of point cloud objects intended for styling operations
+ */
+export abstract class PointCloudObjectCollection {
+  private readonly _changedEvent = new EventTrigger<() => void>();
+
+  abstract getAnnotationIds(): Iterable<number>;
+
```

Comment:
Add TSDOC

Reply:
Added some here and in some more of the public-facing part of the code of `CognitePointCloudModel`